index.js

```diff
@@ -0,0 +1,22 @@
+
+document.addEventListener("DOMContentLoaded", ()=>{
+    axios.get("http://localhost:3000/people").then(res =>{
+        debugger
+        let ul = document.querySelector("ul")
+        res.data.results.forEach(person =>{ 
+            let li = document.createElement("li")
+<<<<<<< HEAD
```

Comment:
When you see `<<<<<<< HEAD` ,  `=======` and `>>>>>>>` it means theres a merge conflict. 

Anything between`<<<` and `===` means thats what was there before and `=====` and `>>>` is what is there now, so you have to decide which you want to keep so you have to delete one segment or the other.